app/controllers/preupgrade_reports_controller.rb

```diff
@@ -0,0 +1,8 @@
+class PreupgradeReportsController < ::Api::V2::BaseController
+  include Utils
+
+  def index
+    @preupgrade_reports = resource_scope.search_for(*search_options)
```

Comment:
Wouldn't `resource_scope_for_index` work here?

Reply:
It would, but it would give me paginated results while I want all of them. Workaround would be to pass very large per_page number and I'd like to avoid that hack.

Reply:
Fair enough